modules/opensearch/includes/opensearch.search.inc

```diff
@@ -601,3 +601,46 @@ function opensearch_search_object_ids($ids) {
   }
   return FALSE;
 }
+
+function opensearch_search_autocomplete($query) {
+  $items = array();
+  $rest_url = variable_get('opensearch_search_autocomplete_suggestion_url', '');
+  $method = variable_get('ting_search_autocomplete_method', 'facets');
+  $settings = ting_search_autocomplete_settings();
```

Comment:
This settings function is probably specific to `opensearch`. Remember to move and rename it.